.Documentation/articles/pro/index.md

```diff
@@ -0,0 +1,7 @@
+# Overview
```

Comment:
Is this file needed? How do you get to articles/pro/index ? Isn't pro more an extension of core?

Reply:
rewrote it, forgot to "properly" fill it. Dont know if we should keep it that way. Are there any better ideas how to place the CreatorPro stuff in the documentation?

Reply:
@tomwim there is a new section in the articles called `Creator Pro` or something like that.